examples/client/Locomotion/src/context/newRideContext/index.tsx

```diff
@@ -289,20 +289,32 @@ const RidePageContextProvider = ({ children }: {
     await StorageService.delete('lastRideId');
   };
 
+  const refreshBusinessAccountIdFromStorage = async () => {
+    const storedId = await StorageService.get('lastBusinessAccountId');
+    if (storedId && storedId !== PAYMENT_MODES.PERSONAL) {
+      setBusinessAccountId(storedId);
+    } else {
+      setBusinessAccountId(null);
+    }
```

Comment:
```suggestion
    const notPersonalPaymentMode = !storedId || storedId !== PAYMENT_MODES.PERSONAL;
    setBusinessAccountId(notPersonalPaymentMode ? null : storedId);
```
Nit: I think it's more clear that way